Models/SpriteModel.cpp

```diff
@@ -27,6 +27,9 @@ QVariant SpriteModel::data(const QModelIndex& index, int role) const {
     return QVariant(QColor(Qt::yellow));
   } else if (role == Qt::SizeHintRole) {
     // Don't load image we just need size
+    if (index.row() < 0) {
```

Comment:
Yes that code prevents the crash as well. I've pushed a new commit with those changes.

Reply:
Ok, that's perfect daz, we should always do this anyway, it does appear in official Qt examples of custom models. Now, let me tell you something I found, I actually discovered where the invalid model index is coming from.
https://github.com/enigma-dev/RadialGM/blob/8fdcf5fd5c018c18fc782a91c9b0366a422a3fee/Editors/SpriteEditor.cpp#L15
If we skip this change, and comment that line, the crash also goes away for me. I assume we are constructing an invalid model index there to request the size. I'll look at that after merging this. I think what I will ask Josh/Rusky if we want to do is assert these model indexes are valid only in debug mode. I am worried that always checking they are valid like this will allow us to write buggy/incorrect code, like the line that actually crashes here, and it will go unnoticed because we are constantly validating the model indexes. Feel free to offer your opinion on this too.

Reply:
I've filed #62 to ensure we have a discussion about this.